enterprise/tpa_pipeline.py

```diff
@@ -142,7 +142,7 @@ def redirect_to_consent():
     if not enterprise_customer.requests_data_sharing_consent:
         # This enterprise customer attached to this pipeline element does not request data sharing consent;
         # proceed with the creation of a link between the user and the enterprise customer, then exit.
-        EnterpriseCustomerUser.objects.create(
+        EnterpriseCustomerUser.objects.get_or_create(
```

Comment:
@mattdrayer yeah. Fixing was quick but correctly configuring tpa-saml on my local machine took 2 days and thanks to @saleem-latif I got it working today.  :stuck_out_tongue_closed_eyes: